startupscript/git-setup.sh

```diff
@@ -46,5 +46,14 @@ ${RUN_AS_LOGIN_USER} "mkdir -p '${WORKBENCH_GIT_REPOS_DIR}'"
 # to the git references, the corresponding git repo cloning will be skipped.
 # Keep this as last thing in script. There will be integration test for git cloning (PF-1660). If this is last thing, then
 # integration test will ensure that everything in script worked.
-${RUN_AS_LOGIN_USER} "cd '${WORKBENCH_GIT_REPOS_DIR}' && wb git clone --all"
-
+pushd "${WORKBENCH_GIT_REPOS_DIR}" || exit 1
+${RUN_AS_LOGIN_USER} "wb resource list --type=GIT_REPO --format json" | \
+  jq -c .[] | \
```

Comment:
oh is git clone --all broken

Reply:
I added some notes about this in the PR description.  It's not really "broken"; looking at the CLI code it's working-as-intended, as the CLI treats git as a "passthrough" command and all passthrough commands require that Application Default Credentials are available.  Git doesn't actually make use of ADC, but we would have to special-case it not to behave this way.

So we could change the CLI behavior, but it seems like a design change rather than a bugfix, so I would rather not gate AWS support on this.  I think there's probably scope for re-visiting the whole "passthrough command" design in the CLI, but it's not trivial.

I'd also rather not branch the script when this logic works for both clouds.  WDYT?

Reply:
i see.  ya that sounds good to me to do it separately

so all the generic toolings are broken for aws instance?

Reply:
Yeah, though other than `git`, I'm not sure why a VM user (AWS or GCP) would run any of them.  They could app-default login with `gcloud` on AWS to get access to GCP resources, without having to wrap them in `wb` calls.

I think the passthrough apps are mainly for devs to run tools as pet SA's on their laptops?  That's why I'm really hesitant to mess with the app behavior without more investigation; I don't understand if/how users are using them for commands other than git.

Reply:
Created [BENCH-3026](https://verily.atlassian.net/browse/BENCH-3026).